lib/index.js

```diff
@@ -1,55 +1,88 @@
 'use strict';
 
-var a = {}; // re-useable Object
+var AWS = require('aws-sdk');
+var AwsHelper = {
+  _AWS: AWS,
+  _context: null,
+  _Lambda: null,
+  _SNS: null,
+  _DynamoDB: null,
+  region: null,
+  version: null,
+  account: null,
+  Lambda: {},
+  SNS: {},
+  DynamoDB: {}
+};
 
-function extractParams (invokedFunctionArn) {
-  // console.log(' >> ', invokedFunctionArn);
+// initialisation method (Optional)
+function init (context) {
+  AwsHelper._context = context;
+  AwsHelper._Lambda = null;
+  AwsHelper._SNS = null;
+  AwsHelper._DynamoDB = null;
+  AwsHelper._parseInvokedFunctionArn(context.invokedFunctionArn);
+  AwsHelper._AWS.config.region = AwsHelper.region;
+  return AwsHelper;
+}
+
+AwsHelper._parseInvokedFunctionArn = function (invokedFunctionArn) {
+  if (!invokedFunctionArn) throw new Error('Unexpected invokedFunctionArn format: ' + invokedFunctionArn);
```

Comment:
Would it helpful to change the error here to 'invokedFunctionArn needs to be specified +invokedFunctionArn '?